src/ja/server/database/types/work_machine.py

```diff
@@ -99,6 +99,14 @@ def online(self) -> bool:
           False otherwise.
         """
 
+    @property
```

Comment:
I would prefer to see the job machine as a state (ONLINE, RETIRED, OFFLINE) and then removing the online attribute.
Otherwise we have 2 boolean flags but not all of the 4 states are valid (i.e not retired but offline)